integrations/pinterest-tag/test/index.test.js

```diff
@@ -63,4 +63,27 @@ describe('Pinterest', function () {
       analytics.load(pinterest, done)
     })
   })
+
+  describe('after loading', function () {
+    beforeEach(function (done) {
+      analytics.once('ready', done)
+      analytics.initialize()
+      analytics.stub(pinterest, 'load')
+    })
+
+    describe('#identify', function () {
+      beforeEach(function () {
+        analytics.stub(window, 'Pinterest')
+      })
+
+      it('should not fire the Pinterest pixel tag', function () {
+        analytics.identify()
+	analytics.didNotCall(pinterest.load)
+      })
+      it('should push Segment email to Pinterest Enhanced Match', function () {
+        analytics.identify('123', { email: 'prakash@segment.com'})
+        analytics.called(pinterest.load)
```

Comment:
let's change this to `analytics.called(pinterest.load, 2620795819800, {em: "prakash@segment.com"})`